template.json

```diff
@@ -39,6 +39,43 @@
                 }
               }
             },
+            "geographic_hierarchy": {
+              "type": "nested",
+              "properties": {
+                "area_types": {
+                  "type": "nested",
+                  "properties": {
+                    "id": {
+                      "type": "string",
+                      "store": true,
+                      "term_vector": "with_positions_offsets",
+                      "analyzer": "snowball"
+                    },
+                    "level": {
+                      "type": "long"
+                    },
+                    "title": {
+                      "type": "string",
+                      "store": true,
+                      "term_vector": "with_positions_offsets",
+                      "analyzer": "snowball"
+                    }
+                  }
+                },
+                "id": {
+                  "type": "string",
+                  "store": true,
+                  "term_vector": "with_positions_offsets",
+                  "analyzer": "snowball"
+                },
+                "title": {
+                  "type": "string",
+                  "store": true,
+                  "term_vector": "with_positions_offsets",
+                  "analyzer": "snowball"
```

Comment:
So we want to search on both the `hierarchy`'s `id` and `title` as well as the `areaType`'s `id` and `title`? Just want to know why we need both and if it's sensible and not an overkill, we'll keep both.